src/content/docs/network-performance-monitoring/setup-performance-monitoring/snmp-performance-monitoring.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Set up SNMP monitoring
+tags:
+  - Integrations
+  - Network performance monitoring
+  - Installation
+  - Setup
+metaDescription: Set up SNMP monitoring.
+---
+
+Set up your network devices so they send network data to New Relic One.
+
+## Prerequisites [#prerequisites]
+
+- A **New Relic account ID**. Read how to [find your account ID](https://docs.newrelic.com/docs/accounts/accounts-billing/account-setup/account-id/).
+- An **Insights Insert Key**. Read how to [generate an Insights insert key](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/#insights-insert-key).
+- [Docker](https://docs.docker.com/engine/install/) installed in a Linux host.
+
+## Set up SNMP monitoring in New Relic One [#setup-snmp-monitoring]
+
+1. In your local machine, from a Linux host with Docker installed, download the **ktranslate** image from [dockerhub](https://hub.docker.com/) by running 
+  ```shell
+  docker pull kentik/ktranslate:v2
+  ```
+2. Copy the `snmp-base.yaml` file to the local `$HOME` directory of your Docker user, and discard the container by running
+    ```shell
+    cd .
+    id=$(docker create kentik/ktranslate:v2)
+    docker cp $id:/etc/ktranslate/snmp-base.yaml .
+    docker rm -v $id
+    ```
+3. Edit the `snmp-base.yaml` file and define the `cidrs` and `default_communities` variables to appropriate values for your network.
+4. Optionally, you can launch a discovery run on SNMP devices. This helps you ensure your devices are ready to be monitored. This discovery run is based on the values you defined for the `cidrs` and `default_communities` variables by running
+    ```shell
+    docker run -ti --name ktranslate-discovery --rm --net=host \
+    -v `pwd`/snmp-base.yaml:/snmp-base.yaml \
+    kentik/ktranslate:v2 \
+        -snmp /snmp-base.yaml \
+        -log_level info \
+        -snmp_discovery=true
+    ```
+  After the discovery run finishes, you should see an output similar to the following:
+  ```shell
+  >[Info] KTranslate Adding 3 new snmp devices to the config, 0 replaced from 3
+  # In this example, the discovery run found 3 new SNMP devices.
+  ```
+  The discovered devices are listed in the `snmp-base.yaml` file. By default, only the `IF-MIB` mib is polled. You can manually add other mibs here if your devices support them. 
+5. Run `ktranslate` to poll target devices. Use the following example:
+
+  <Callout variant="important">
```

Comment:
I'd move the callout after the example.

Reply:
The SME suggested to add it before to avoid users running the command before seeing the information and running into an error.